pf-keys/definitions/geom.yaml

```diff
@@ -1480,6 +1488,151 @@ TopoSlopesY:
         domains:
           DoubleValue:
 
+# -----------------------------------------------------------------------------
+# ChannelWidth
+# -----------------------------------------------------------------------------
+# ChannelWidth missing from manual and grep
+
+# TODO: Change the documentation to match with channel width, also add handlers if needed? (Not sure if they are needed)
```

Comment:
Not sure what first part of this TODO is saying.

Reply:
Handlers were also added, this comment should have been removed.